test/sandbox.test.js

```diff
@@ -0,0 +1,53 @@
+import PlanGateway from '../lib/gateways/plan-gateway';
```

Comment:
Was this file intentionally committed?

Reply:
It will be removed but I'd rather not mess about getting dynamo local working every time I need to spike haha